sections/raw.js

```diff
@@ -82,7 +82,7 @@ export default [
     type: "runnable",
     content: `
       const myArray = [1,2,3]
-      knex.raw('select * from users where id in (' + myArray.map(_ => '?').join(',') + ')', [...myArray]);
+      ;knex.raw('select * from users where id in (' + myArray.map(_ => '?').join(',') + ')', [...myArray]);
```

Comment:
Shouldn't `;` be on the previous line?

Reply:
Docs are using semicolon-free style and in semicolon-free stile all lines that start from a `[` or `(` must start from a semicolon. This is important because ending only the line before `[` or `(` with a semicolon is error-prone because in this case, a harmless movement of lines may break the code